.github/workflows/kbs-build-and-push.yaml

```diff
@@ -93,11 +93,35 @@ jobs:
     - name: Publish Multi-Arch ${{ matrix.image }} image
       run: |
         commit_sha=${{ github.sha }}
-        docker manifest create ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha} \
-          --amend ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}-x86_64 \
-          --amend ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}-s390x
-        docker manifest push ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}
-        docker manifest create ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest \
-          --amend ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest-x86_64 \
-          --amend ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest-s390x
-        docker manifest push ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest
+        docker manifest create "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}" \
+          --amend "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}-x86_64" \
+          --amend "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}-s390x"
+        docker manifest push "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:${commit_sha}"
+        docker manifest create "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest" \
+          --amend "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest-x86_64" \
+          --amend "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest-s390x"
+        docker manifest push "ghcr.io/confidential-containers/staged-images/${{ matrix.image }}:latest"
+
+  build_and_push_ita:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GHCR Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think these parts are duplicated. Could we only add the `docker build ... --push` partsright under the existing parts?

Reply:
I agree but ITA for now shouldn't be automatically built for arch different than x86_64 and with current approach is not clear for me how to approach this without if statements and running workflow twice (for both arch types)

Reply:
Excluded one config and added to exiting approach